controllers/config/crd/bases/config.porch.kpt.dev_fleetmembershipbindings.yaml

```diff
@@ -1,24 +1,9 @@
-# Copyright 2023 The kpt and Nephio Authors
```

Comment:
Why are these getting removed?

Reply:
Good spot. Not sure how they were getting added before. I have update the controller-gen directives to add the header now.